### PR TITLE
(#198) Add formula 5.7 from NEN-EN-1993.5:2008 

### LINKED ISSUE #198: [✨ Feature request]: Add formula 5.7 from NEN-EN 1993-5
### Description

Add formula 5.7 from NEN-EN 1993-5

### Acceptance Criteria

- Formula 
- Latex
- Unit-tests
- Docstrings 

### Dependencies

_No response_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by Klaassicking**
## Description

Add formula 5.7 from NEN-EN-1993.5:2008 

$V_{b,Rd} = \frac{\left(h - t_f \right) t_w f_{bv}}{\gamma_{M0}}$

Fixes #198 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by egarciamendez — APPROVED** [review]
Already fixed according to the other formulas. 🚀

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_7.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                r"V_{b,Rd} = \frac{\left(h - t_f \right) t_w f_{bv}}{\gamma_{M0}} = \frac{(500 - 20) \cdot 10 \cdot 300}{1.1} = 1309.090909090909",
+            ),
+        ],
+    )
+    def test_latex_output(self, representation: str, expected: str) -> None:
+        """Test the latex implementation."""
+        h = 500  # MM
+        t_f = 20  # MM
+        t_w = 10  # MM
+        f_bv = 300  # kN
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_7.py:71]:
```suggestion
        f_bv = 300  # MPA
```

File: tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_7.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+class TestForm5Dot7ShearBucklingResistance:
+    """Validation for formula 5.7 from NEN-EN 1993-5:2008."""
+
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # Example values
+        h = 500  # MM
+        t_f = 20  # MM
+        t_w = 10  # MM
+        f_bv = 300  # kN
```
Comment (bro-wi) [inline tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_7.py:18]: MPA

**Review by bro-wi — CHANGES_REQUESTED** [review]

File: blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_7.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        gamma_m_0: float,
+    ) -> KN:
+        """Evaluates the formula for shear buckling resistance."""
+        raise_if_less_or_equal_to_zero(
+            h=h,
+            t_f=t_f,
+            t_w=t_w,
+            f_bv=f_bv,
+            gamma_m_0=gamma_m_0,
+        )
```
[the remark this answers is not in the document]
Comment (egarciamendez) [reply blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_7.py:67]: How about a plain old ValueError for this one?

**Pull request merged**